### PR TITLE
Add rejected column and blockers

### PR DESCRIPTION
**Pull request by hellobart**
[Finishes #169836792]

### CODE REVIEW

File: src/Settings.js [hunk 1]
```diff
@@ -60,6 +61,28 @@ const Settings = () => {
           </p>
         </div>
       </div>
+
+      <div className="media">
+        <div className="media-left">
+          <label className="switch">
+            <input
+              type="checkbox"
+              checked={rejected ? 'checked' : false}
```
Comment (HenryNguyen5) [inline src/Settings.js]: Shouldnt this be `true`?

Comment (HenryNguyen5) [reply src/Settings.js]: Then we could pass `rejected` as the value

File: src/StoryBlockers.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBan } from '@fortawesome/free-solid-svg-icons';
+
+function StoryBlockers({ blockers }) {
+  if (blockers && blockers.length) {
```
Comment (HenryNguyen5) [inline src/StoryBlockers.js]: Lets check the inverse here and return `null` early, negating the need to have this logic in a nested conditional block

File: src/StoryBlockers.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    return (
+      <div data-testid="story-blockers" className="content is-small">
+        <hr style={{ margin: '0px 0 10px' }} />
+        <span className="has-text-danger">
+          <FontAwesomeIcon icon={faBan} size="1x" /> Blockers:
+        </span>
+
+        <ul style={{ marginTop: '10px' }}>
+          {blockers.map((b, i) => (
+            <li key={i}>
```
Comment (HenryNguyen5) [inline src/StoryBlockers.js]: Lets use a more descriptive key like `blocker-${i}`

File: src/StoryBlockers.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+              <a
+                href={b && b.url}
```
Comment (HenryNguyen5) [inline src/StoryBlockers.js]: Won’t `b` always exist here?

File: src/StoryBlockers.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                className="is-small has-text-grey-dark has-hover-underline"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {b && b.name}
```
Comment (HenryNguyen5) [inline src/StoryBlockers.js]: Wont `b` always exist here?

File: src/StoryBlockers.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+          ))}
+        </ul>
+      </div>
+    );
+  } else {
+    return null;
+  }
+}
+
+export { StoryBlockers };
```
Comment (HenryNguyen5) [inline src/StoryBlockers.js]: We could do an export directly from the function instead

File: src/Story.test.js [hunk 7]
```diff
@@ -142,31 +142,21 @@ describe('Story', () => {
...
-      const { queryByTestId } = renderSubject({
-        blockers: [{ resolved: true }]
-      });
-
-      expect(queryByTestId('blocked-tag')).toBeInTheDocument();
-      expect(queryByTestId('blocked-tag')).not.toBeVisible();
-    });
-
-    it('renders invisible blocked tag if no blockers', () => {
+    it('does not rnender blockers if no blockers', () => {
```
Comment (HenryNguyen5) [inline src/Story.test.js]:
```suggestion
    it('does not render blockers if no blockers exist', () => {
```

File: src/Settings.js [hunk 8]
```diff
@@ -7,6 +7,7 @@ const Settings = () => {
   );
 
   const [slim, setSlim] = useLocalStorage('slim');
+  const [rejected, setRejected] = useLocalStorage('rejected');
```
Comment (HenryNguyen5) [inline src/Settings.js]: What do you think of the variable name `showRejectedColumn`?

File: src/Settings.js (hunk 1 above)
Comment (hellobart) [reply src/Settings.js]: I wanted to keep consistency with other options. Also checked=checked is valid attribute [Living Standard](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute)

File: src/StoryBlockers.js (hunk 6 above)
Comment (hellobart) [reply src/StoryBlockers.js]: Good point but i wanted to keep consistency with other components.

File: src/Settings.js (hunk 8 above)
Comment (hellobart) [reply src/Settings.js]: I used `rejected` to keep consistency with the `slim` option but i agree with you here.

File: src/StoryBlockers.js (hunk 6 above)
Comment (HenryNguyen5) [reply src/StoryBlockers.js]: Ah, makes sense

**Pull request merged**